mplc/contributivity.py

```diff
@@ -938,103 +940,88 @@ def without_replacment_SMC(self, the_scenario, sv_accuracy=0.01, alpha=0.95):
             end = timer()
             self.computation_time_sec = end - start
 
-    def federated_SBS_linear(self, the_scenario):
-        start = timer()
-        logger.info(
-            "# Launching computation of perf. scores of linear "
-            "performance increase compared to previous collective model")
-
-        relative_perf_matrix = self.compute_relative_perf_matrix(the_scenario)
```

Comment:
Same in this file, it looks like it wasn't well rebased, and it erases quite a lot of recently merged material.